drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -1012,6 +1017,12 @@ local function match_modular_profile_thermostat(driver, device)
 
   if #fan_eps > 0 then
     table.insert(main_component_capabilities, capabilities.fanMode.ID)
+    for _, p in ipairs(THERMOSTAT_SUPPORTED_PERCENT_PRODUCTS) do
+      if device.manufacturer_info.vendor_id == p[1] and device.manufacturer_info.product_id == p[2] then
+        table.insert(main_component_capabilities, capabilities.fanSpeedPercent.ID)
```

Comment:
Additional capability support for specific vendor's product (under CN WWST program)